src/store.js

```diff
@@ -13,8 +14,11 @@ const rootReducer = combineReducers({
 });
 const middleware = applyMiddleware(thunkMiddleware);
 
+const persistedLogin = loadLogin();
+
 const store = createStore(
   rootReducer,
+  persistedLogin,
```

Comment:
Will loadLogin always produce a structure that is in sync with the redux state tree? or would you load the config, build the 'seeded' tree elsewhere (to safeguard the structure, and allow other components to get their persisted state as well and seed it into the initial state)